.github/workflows/ci.yaml

```diff
@@ -446,7 +446,7 @@ jobs:
       run: |
           git checkout v${{ matrix.from_version }} --no-overlay -- examples
           git checkout v${{ matrix.from_version }} --no-overlay -- tests/cases
-          git checkout v${{ matrix.from_version }} --no-overlay -- tests/helper_components
+          git checkout v${{ matrix.from_version }} --no-overlay -- tests/helper_components/*.slint
```

Comment:
To avoid losing `helper_components/Cargo.toml` or `helper_components/src/lib.rs` that are needed because the test driver for Rust lists `helper_components` as a dependency:

Missing `Cargo.toml`:
```
error: failed to load manifest for workspace member `/Users/jpnurmi/Projects/rust/slint/tests/driver/rust`

Caused by:
  failed to load manifest for dependency `helper_components`

Caused by:
  failed to read `/Users/jpnurmi/Projects/rust/slint/tests/helper_components/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```

Missing `src/lib.rs`:
```
error: failed to load manifest for workspace member `/Users/jpnurmi/Projects/rust/slint/tests/driver/rust`

Caused by:
  failed to load manifest for dependency `helper_components`

Caused by:
  failed to parse manifest at `/Users/jpnurmi/Projects/rust/slint/tests/helper_components/Cargo.toml`

Caused by:
  no targets specified in the manifest
  either src/lib.rs, src/main.rs, a [lib] section, or [[bin]] section must be present
```